CITING.md

```diff
@@ -0,0 +1,5 @@
+# Citing
+
+Cite HADDOCK3 with the following:
+
+HADDOCK3, Bonvin's Lab, https://github.com/haddocking/haddock3, 2022.
```

Comment:
I guess we should give at least the original HADDOCK publication for the time being. I.e the JACS 2003 one, next to the above line.

Reply:
And may-be also add `Developed with financial support from the Horizon 2020 BioExcel CoE project [823830](https://cordis.europa.eu/project/id/823830))`

Reply:
Done